### ISSUE TITLE
Updating from ELF to RPX

### ISSUE THREAD

**Issue by xhp-creations** [vgmoose/wiiu-hbas#25]
When updating my libretro cores from ELF to RPX builds using HBAS, the old ELF files remain, so they are listed twice in HBL.  Need to delete the old ELF, or give option of doing that.

**Comment by vgmoose** [comment]
I guess the fix for this then could literally be:
```
if new downloaded type is file.rpx:
    - delete file.elf in the app folder
else if new downloaded type is file.elf:
    - delete file.rpx in the app folder
```

**Comment by vgmoose** [comment]
Fixed in f2d08f3d4 by @rw-r-r-0644
